larq_compute_engine/mlir/python/converter.py

```diff
@@ -113,6 +113,12 @@ def convert_keras_model(
             "Using `experimental_default_int8_range` as fallback quantization stats. "
             "This should only be used for latency tests."
         )
+    if hasattr(model, "dtype_policy") and model.dtype_policy.name == "mixed_float16":
```

Comment:
How do you feel about changing this to:
```suggestion
    if hasattr(model, "dtype_policy") and model.dtype_policy.name != "float32":
```
Since `dtype_policy` can also be `bfloat16` in rare cases which I guess is also not supported.